RedisManager closeConnection utáni hívásra nem megfelelő hibával száll el

closeConnection() nem nullozza ki a jedis példányt, így ha close után ráhívunk nem "jedis is not initialized" helyett elég egzotikus hibák jönnek. pl.:

```
Caused by: java.lang.ClassCastException: class java.util.ArrayList cannot be cast to class java.lang.Long (java.util.ArrayList and java.lang.Long are in module java.base of loader 'bootstrap')
	at deployment.ROOT.war//redis.clients.jedis.Connection.getIntegerReply(Connection.java:294)
	at deployment.ROOT.war//redis.clients.jedis.Jedis.expire(Jedis.java:452)
```

Mivel nincs kinullozva így újrahasználni se lehet ugyanazt a managert, mert close után az init nem fog csinálni semmit

--- a/coffee-module/coffee-module-redis/src/main/java/hu/icellmobilsoft/coffee/module/redis/manager/RedisManager.java
+++ b/coffee-module/coffee-module-redis/src/main/java/hu/icellmobilsoft/coffee/module/redis/manager/RedisManager.java
@@ -491,7 +491,9 @@ public class RedisManager {
      */
     public void closeConnection() {
         if (jedis != null) {
-            jedisInstance.destroy(jedis);
+            Jedis destroyable = jedis;
+            jedisInstance.destroy(destroyable);
+            jedis = null;
         }
     }
 

